### PR TITLE
Fix for the title of a loaded module is not visible

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #36965 .

### Summary of Changes

I added fallback from `xhtml` to `html5`.
Improved perfomance check, and removed useles things.

### Testing Instructions
Please follow #36965

Create a cutom module.
Activate the title of this module
Load this module into an article with `{loadposition position-name}`

Go to pluging "Content - Load Modules", set style "Wrapped by div".

View the article page, before and after patch.

Repeat the test for `{loadmodule mod_name,mod title}` placeholder.

### Actual result BEFORE applying this Pull Request
Module displaied without title

### Expected result AFTER applying this Pull Request
Module displaied with title

### Link to documentations
Please select:
- [x] No documentation changes for docs.joomla.org needed
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: plugins/content/loadmodule/loadmodule.php [hunk 1]
```diff
@@ -54,101 +54,101 @@ public function onContentPrepare($context, &$article, &$params, $page = 0)
             return;
         }
 
-        // Simple performance check to determine whether bot should process further
```
Comment (brianteeman) [inline plugins/content/loadmodule/loadmodule.php:57]: is this not still needed?

Comment (Fedik) [reply plugins/content/loadmodule/loadmodule.php:57]: I replaced it with a better check, look at lines: 73, 101, 137

File: plugins/content/loadmodule/loadmodule.xml [hunk 2]
```diff
@@ -23,14 +23,15 @@
...
 					type="list"
 					label="PLG_LOADMODULE_FIELD_STYLE_LABEL"
-					default="table"
+					default="none"
 					validate="options"
 					>
+					<option value="none">PLG_LOADMODULE_FIELD_VALUE_RAW</option>
+					<option value="html5">PLG_LOADMODULE_FIELD_VALUE_DIVS</option>
 					<option value="table">PLG_LOADMODULE_FIELD_VALUE_TABLE</option>
+					<!-- TODO: following styles doesn't exist in default installation and can be removed in Joomla 5 -->
```
Comment (richard67) [inline plugins/content/loadmodule/loadmodule.xml]:
```suggestion
					<!-- TODO: following styles don't exist in default installation and can be removed in Joomla 5 -->
```

Comment (brianteeman) [reply plugins/content/loadmodule/loadmodule.xml]:
```suggestion
					<!-- @TODO: The following styles don't exist in default installation and can be removed in Joomla 5 -->
```

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 20af46a6a32b57b8c4c5b1c331ea57285270d4d1<br /><br />See comment<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39839">issues.joomla.org/tracker/joomla-cms/39839</a>.</sub>

**Comment by sandewt** [comment]
Before:
issue-36965-before


After:
issue-36965-after

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 20af46a6a32b57b8c4c5b1c331ea57285270d4d1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39839">issues.joomla.org/tracker/joomla-cms/39839</a>.</sub>

**Comment by sandewt** [comment]
I ran **Test 2** without `mod_` as a prefix !!!

In retrospect, this did not seem quite right to me.

To be sure, I therefore did a **Test 3** with `mod_custom`.
See the picture:
issue-36965-after-2


@Fedik Please your opinion? Did you mean it like that?

**Comment by Fedik** [comment]
@sandewt both is valid, `{loadmodule mod_custom,mod title}` and `{loadmodule custom,mod title}` should work the same

**Comment by sandewt** [comment]
@Fedik Thanks for the confirmation.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39839">issues.joomla.org/tracker/joomla-cms/39839</a>.</sub>

**Comment by roland-d** [comment]
Thank you

**Pull request merged**